frontend/src/styles/_compose.scss

```
@@ -84,59 +88,81 @@
   padding: 18px 20px 22px;
   background: var(--surface);
   border-top: 1px solid var(--border);
+  // Opening a reply scrolls this row into view. The top bar is sticky and 64px
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)